crud/common/types.lua

```diff
@@ -0,0 +1,23 @@
+local types = {}
+
+local lt_by_key_type = {
```

Comment:
I think such approach could break decimal support

Reply:
If it works, it will continue working. If a comparator is missing, it falls back to a common one.